glpk/meta.yaml

```diff
@@ -17,6 +17,16 @@ requirements:
   build:
     - gmp      [not win]
 
+test:
```

Comment:
I get the same error with cython as a runtime dependency as well,
unfortunately.

I've worked around the issue by pre-generating the C file and including it.

On Wed, Feb 17, 2016 at 7:27 AM, Mike Sarahan <notifications@github.com>
wrote:

> In glpk/meta.yaml
> <https://github.com/conda/conda-recipes/pull/595#discussion_r53179439>:
>
> > @@ -17,6 +17,16 @@ requirements:
> >    build:
> >      - gmp      [not win]
> >
> > +test:
>
> Ugh, we have some issues with test-only dependencies. If you don't mind,
> please add it as a runtime dependency also, so that it gets installed
> correctly. I don't want our test-dependency issue to hold up your PR. Add a
> comment about this to the recipe, if you'd like.
>
> Ping @kalefranz <https://github.com/kalefranz> @mcg1969
> <https://github.com/mcg1969> - if either of you run into code that might
> address this problem, let me know.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/conda/conda-recipes/pull/595/files#r53179439>.
>